github-git/social-open-source.md

```diff
@@ -0,0 +1,158 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+myst_html_meta:
+  "title": "The social side of open source: Engage, build trust & support contributors"
+  "description lang=en": "A guide to the social elements that will make contributing to open source more impactful, valuable, and successful."
+  "keywords": "GitHub, OpenSource, beginner-friendly"
+  "property=og:locale": "en_US"
+---
+:::{todo}
+Link to relevant sections that cover each of the topics below
+:::
+
+# The social side of open source: Engage, build trust & support contributors  
+
+:::{figure} /images/github/open-source-social-network.png
+:alt: A digital illustration visualizing open-source collaboration on GitHub. At the center, a “Main Repo” connects to contributors performing key actions: forking (branching icon), submitting pull requests (PRs) (merge arrows), reviewing code (magnifying glass over documents), and engaging in discussions (speech bubbles and coding icon). The pyOpenSci logo appears in the bottom right, reinforcing the theme of community-driven contributions.
+
+
+:::
+
+## Why social etiquette matters in open source  
+
+Contributing to open source isn’t just about code—it’s about **collaborating with people you may never meet in real life**. Unlike traditional workplaces, GitHub interactions happen asynchronously, across time zones, and often with volunteers.  
+
+This means **how you communicate is just as important as what you contribute**.  
+
+This lesson covers three key aspects of open source etiquette:  
+
+**1️⃣ Engage thoughtfully in open source communities**  
+Learn how to interact professionally in issues, PRs, and discussions to make collaboration smooth and efficient.  
+
+**2️⃣ Build trust through transparent collaboration**  
+Trust helps your contributions get reviewed and merged faster. You’ll learn how to show maintainers you’re a reliable contributor.  
+
+**3️⃣ Don't bite off more than you can chew**
+
+
+:::{admonition} For maintainers
+:class: tip
```

Comment:
or you can just leave a comment. If you leave multiple comments it's nice to do it as a review vs single comments. this just means all of the comments will appear at one time when you are complete with the review! it means fewer notifications to the repo owners and followers :)